main.go

```diff
@@ -0,0 +1,73 @@
+package main
```

Comment:
Missing SPDX license declaration. (Feedback applies to every Go file)

Example:

```suggestion
// SPDX-License-Identifier: Apache-2.0
// Copyright Authors of Cilium

package main
```

Perhaps @xmulligan could speak to whether we should explictly put an additional line for `// Copyright Isovalent, Inc`. Alternative would be to highlight in the commit message that the initial versions are all copyright-assigned Isovalent. As I understand the `Authors of Cilium` notation is intended to reflect that the copyright is shared amongst the respective authors. As a project donated from Isovalent, I assume that Isovalent holds the full copyright for all existing files.